pkg/network/admitter/slirp_test.go

```diff
@@ -52,17 +53,16 @@ var _ = Describe("Validate interface with SLIRP binding", func() {
 	})
 
 	It("should be rejected without a pod network", func() {
-		vmi := v1.VirtualMachineInstance{}
-		vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{{
-			Name: "default",
-			InterfaceBindingMethod: v1.InterfaceBindingMethod{
-				DeprecatedSlirp: &v1.DeprecatedInterfaceSlirp{},
-			},
-		}}
-		vmi.Spec.Networks = []v1.Network{{
-			Name:          "default",
-			NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "net"}},
-		}}
+		vmi := libvmi.New(
+			libvmi.WithInterface(v1.Interface{
+				Name:                   "default",
+				InterfaceBindingMethod: v1.InterfaceBindingMethod{DeprecatedSlirp: &v1.DeprecatedInterfaceSlirp{}},
+			}),
+			libvmi.WithNetwork(&v1.Network{
+				Name:          "default",
+				NetworkSource: v1.NetworkSource{Multus: &v1.MultusNetwork{NetworkName: "net"}},
+			}),
```

Comment:
I think you can use here `libvmi.WithNetwork(libvmi.MultusNetwork("default", "net"))`